src/logic/progress_codes.h

```diff
@@ -50,5 +53,7 @@ enum class ProgressCode : uint_fast8_t {
     HWTestDisplay, // P35
     ErrHwTestFailed, // P36
 
+    _cnt,
```

Comment:
:pray: add a comment describing the fact, that _cnt shall be kept pre-last just before the `Empty` item